addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -2590,6 +2593,9 @@ const Many2oneUserValueWidget = SelectUserValueWidget.extend({
         });
 
         await Promise.all(records.slice(0, this.options.limit).map(async record => {
+            // In case of race between updates of the list, make sure to always
+            // start with an empty list of suggestions.
+            clearSuggestions();
```

Comment:
Why can't we just clear the list here only? (once we are ready to update following the RPC?)

Reply:
Since the content of the method itself is sequential because of the `await`s, I reordered as follows:
- await rpc
- clear the old list
- await render buttons based on rpc result